katz/src/main/kotlin/katz/Id.kt

```diff
@@ -0,0 +1,31 @@
+package katz
+
+class Id<out A>(val value: A): HK<Id.F, A> {
+
+    class F
```

Comment:
It seems to compile with object too in my case. But what are the advantages of declaring it as an object? It will cause the existence of an instance and we probably do not need any.